source/github-helpers/index.ts

```diff
@@ -53,6 +53,10 @@ export const getCurrentBranchFromFeedLink = (): string | undefined => {
 };
 
 export const getCurrentBranchFromFindLink = async (): Promise<string | undefined> => {
+	if (pageDetect.isPRFiles()) {
+		return;
+	}
```

Comment:
Let’s add a comment explaining why. I feel that a lot of code could be explained a little better. Not what it’s doing, but *why* some choices were made